test/fixtures/dep-pkg-1/hooks/install

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env pwsh
```

Comment:
:thinking:  Will this bite us in the future? I believe this works because we rely (in Linux) on there being a system `/usr/bin/env` and binlinking powershell in our setup scripts.  

If we changed this to `#!{{pkgPathFor "core/powershell"}}/bin/pwsh`  and added a dependency on `core/powershell`, we'd be protecting ourselves from test environment changes, at minimal cost. `core/powershell` _should_ already be on disk, and if it isn't only the first build would eat the download/install cost, at least on Linux. Would it be the same on Windows?

This isn't a "must change" or a blocker to merging, just one of the things I think about when I see packages relying on system paths.

Reply:
good point. We might as well address all the hooks that do this (not a ton). filed https://github.com/habitat-sh/habitat/issues/7573